Description:
 Add several header includes to be able to link with system copy of AGG.
 See Debian bug #575620.
Author: Jakub Wilk <jwilk@debian.org>
Forwarded: not-needed
Last-Update: 2010-02-07

--- a/codecs/agg_svg_path_renderer.hh
+++ b/codecs/agg_svg_path_renderer.hh
@@ -19,6 +19,9 @@
 #ifndef AGG_SVG_PATH_RENDERER_INCLUDED
 #define AGG_SVG_PATH_RENDERER_INCLUDED
 
+#include <limits.h>
+#include <stdlib.h>
+
 #include "agg_path_storage.h"
 #include "agg_conv_transform.h"
 #include "agg_conv_stroke.h"
--- a/codecs/svg.cc
+++ b/codecs/svg.cc
@@ -19,6 +19,9 @@
  * Copyright (C) 2002-2005 Maxim Shemanarev (http://www.antigrain.com)
  */
 
+#include <limits.h>
+#include <stdlib.h>
+
 #include "agg_basics.h"
 #include "agg_rendering_buffer.h"
 #include "agg_rasterizer_scanline_aa.h"
--- a/lib/agg.hh
+++ b/lib/agg.hh
@@ -16,6 +16,8 @@
  * copyright holder ExactCODE GmbH Germany.
  */
 
+#include <limits.h>
+
 #include <agg_pixfmt_rgb.h>
 
 #include <agg_rendering_buffer.h>
